### PR TITLE
Fix positioning of create and close buttons

### PR DESCRIPTION
**Pull request by mazameli**
## Before

image

## After

image

### CODE REVIEW

File: frontend/src/metabase/sharing/components/SharingSidebar.jsx [hunk 1]
```diff
@@ -512,22 +512,24 @@ class SharingSidebar extends React.Component {
...
-            </Tooltip>
-            <Tooltip tooltip={t`Close`}>
-              <Icon
-                name="close"
-                className="text-brand bg-light-hover rounded p1 cursor-pointer"
-                size={20}
-                onClick={onCancel}
-              />
-            </Tooltip>
+            <div className="flex align-center">
```
Comment (kdoh) [inline frontend/src/metabase/sharing/components/SharingSidebar.jsx:515]:
```suggestion
            <Flex align="center">
```

File: frontend/src/metabase/sharing/components/SharingSidebar.jsx [hunk 2]
```diff
@@ -512,22 +512,24 @@ class SharingSidebar extends React.Component {
...
+              </Tooltip>
+              <Tooltip tooltip={t`Close`}>
+                <Icon
+                  name="close"
+                  className="text-light bg-light-hover rounded p1 cursor-pointer"
+                  size={22}
+                  onClick={onCancel}
+                />
+              </Tooltip>
+            </div>
```
Comment (kdoh) [inline frontend/src/metabase/sharing/components/SharingSidebar.jsx:532]:
```suggestion
            </Flex>
```

**Review by kdoh — APPROVED** [review]
Looks fine, I left a tiny suggestion about using the `Flex` component on the wrapper that I'll just go ahead and commit via the suggestion UI.

File: frontend/src/metabase/sharing/components/SharingSidebar.jsx (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/sharing/components/SharingSidebar.jsx]: Ah, right, I keep forgetting about `<Flex>`. Thanks!

**Pull request merged**